src/components/Login/index.scss

```diff
@@ -0,0 +1,19 @@
+.login-form-container{
+    background:white;
+    margin:40px auto;
+    display:inline-block;
+    width:525px;
+    height:665px;
+    box-shadow: .5px 0 5px #9e9e9e;
+}
+.login-logo{
+    margin-top:9rem;
+    margin-bottom: 1.5rem;
+}
+.login-form-body{
+    text-align: left;
+}
+.login-button-container{
+    padding-top: 25px;
+    text-align: center;
+}
```

Comment:
Newline here and let's add new lines between each of the style {} to add more spacing
```
.class1 {
  ...
}

.class2 {
   ....
}
```
Also the spacing between properties and their values can be adjusted. i.e. background: white; so the spacing is consistent. Another thing to be consistent is you can alphabetize your style rules so it's not a blob of styles or you can sort of group them more by like layout, dimensions, font, background, etc.

Reply:
Are you using VSCode and do you have the prettie and eslint plugins installed? it should let you save and edit things on the fly so it would fix the indentation, spacing, and eveyrthing to be consistent